reddit_liveupdate/lib/events.py

```diff
@@ -0,0 +1,103 @@
+from r2.lib.eventcollector import (
+    EventQueue,
+    Event,
+    squelch_exceptions,
+    _datetime_to_millis,
+)
+from r2.lib.utils import sampled
+
+
+class LiveUpdateEvent(Event):
+    def __init__(self, lu_event, **kw):
+        kw.setdefault("topic", "live_thread_events")
+        super(LiveUpdateEvent, self).__init__(**kw)
+
+        self.add("liveevent_id", lu_event._id)
```

Comment:
worth just calling it `event_id` etc? the topic's already `live_thread_events` so that should be unambiguous. then we get to avoid the adjacent `ee` :P